### PR TITLE
Add Status.csv to EP Perf Tool

### PR DESCRIPTION
**Pull request by oliviajain**
**Description**: Output Status.csv - Displays each model's pass/fail status for each EP

**Motivation and Context**
- Update model status frequently to catch breaking changes.

### CODE REVIEW

**Comment by jywu-msft** [comment]
@chilo-ms can you take a look?

File: onnxruntime/python/tools/tensorrt/perf/perf.sh [hunk 1]
```diff
@@ -5,49 +5,64 @@ FAIL_MODEL_FILE=".fail_model_map"
...
 
-# symbolic shape info
+# files to download info
 SYMBOLIC_SHAPE_INFER="symbolic_shape_infer.py"
 SYMBOLIC_SHAPE_INFER_LINK="https://raw.githubusercontent.com/microsoft/onnxruntime/master/onnxruntime/python/tools/symbolic_shape_infer.py"
+FLOAT_16="float16.py"
+FLOAT_16_LINK="https://raw.githubusercontent.com/microsoft/onnxconverter-common/master/onnxconverter_common/float16.py"
+
+# root working directory 
+DEFAULT_DIR="/home/olivia/build_cuda/perf/"
```
Comment (jywu-msft) [inline onnxruntime/python/tools/tensorrt/perf/perf.sh:15]: this doesn't seem to be a good default  (should be a generic location and not for specific user)

File: onnxruntime/python/tools/tensorrt/perf/perf.sh [hunk 2]
```diff
@@ -5,49 +5,64 @@ FAIL_MODEL_FILE=".fail_model_map"
...
-    python3 benchmark_wrapper.py -r validate -m $MODEL_LIST -o result/"$1"
-    python3 benchmark_wrapper.py -r benchmark -i random -t 10 -m $MODEL_LIST -o result/"$1"
+    python3 benchmark_wrapper.py -d $DEFAULT_DIR -r validate -m $MODEL_LIST -o result/"$1"
+    python3 benchmark_wrapper.py -d $DEFAULT_DIR -r benchmark -i random -t 10 -m $MODEL_LIST -o result/"$1"
 fi
 
+# Test models 
+if [ "$1" == "models" ]
+then
+    MODEL_LIST="/home/olivia/build_cuda/perf/bertsquad.json"
```
Comment (jywu-msft) [inline onnxruntime/python/tools/tensorrt/perf/perf.sh:64]: avoid /home/olivia

File: onnxruntime/python/tools/tensorrt/perf/benchmark.py [hunk 3]
```diff
@@ -1079,6 +1075,9 @@ def run_onnxruntime(args, models):
                 if ep not in validation_exemption:
                     try:
                         ort_outputs = inference_ort_and_get_prediction(name, sess, inputs)
+                        #logger.info("COMPARE OUTPUTS Ref, Ort")
```
Comment (jywu-msft) [inline onnxruntime/python/tools/tensorrt/perf/benchmark.py:1078]: what are these commented out lines?

**Comment by jywu-msft** [comment]
@chilo-ms , please take a look.

File: onnxruntime/python/tools/tensorrt/perf/benchmark.py (hunk 3 above)
Comment (oliviajain) [reply onnxruntime/python/tools/tensorrt/perf/benchmark.py:1078]: I was using it to investigate the accuracy issues. I can delete them for this PR.

**Review by jywu-msft — APPROVED** [review]

**Review by chilo-ms — APPROVED** [review]

**Pull request merged**